crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -204,6 +214,8 @@ def restore_chef
           raise "Restore failed"
         end
 
+        # allow enable dns after restore.
+        disable_dns_path.delete if disable_dns_path.exist?
```

Comment:
@MaximilianMeister : The idea was to delete the file before the chef-client is run. Thats why I put it outside cleanup().